e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -49,7 +49,9 @@ describe("scenarios > collection defaults", () => {
       });
 
       popover().within(() => {
-        cy.findByText(`Collection ${COLLECTIONS_COUNT}`).click();
+        cy.findByText(`Collection ${COLLECTIONS_COUNT}`).click({
+          waitForAnimations: false,
```

Comment:
Shorter term to unblock backport PR from merging  I'd say this will suffice, but I think we should actually fix the root cause which is why we can scroll even if we hit the end of the page when we open the collection selector in the first place.

If you try to do the same when only the new collection modal is open, but don't click the collection selector, there's no way to scroll beyond the end of the page. Only if you click the collection selector then you can keep scrolling down infinitely.
<img width="834" alt="image" src="https://user-images.githubusercontent.com/1937582/224986886-321fdcf0-94ca-4f7e-bd22-8c5cff5f52dd.png">

For this reason, I think we should add a comment explaining this, so we know it's something we should fix, what do you think?